components/form/overview.md

```diff
@@ -241,40 +192,55 @@ The Form also uses the the following attributes from the model:
+### Form Layout Customization
 
-* `ValidationMessageType` - `enum` - define the validation message type for the Form. See the [Validation]({%slug form-validation%}) article for more information.
+The Blazor Form exposes multiple parameters that allow you to customize its layout:
 
-* `FormItems` - `RenderFragment` - allows you to use custom editors. See the [FormItems]({%slug form-formitems%}) article for more information. When a custom editor is used, no built-in editors will be generated.
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
 
-* `FormButtons` - `RenderFragment` - allows you to add custom buttons to the Form. You can use the `FormButtons` tag to [add a Clear button to the Form]({%slug form-formitems-buttons%}). If the `FormButtons` tag is defined there will be no default buttons in the Form. 
+| Parameter | Type and Default value | Description |
+|-----------|------------------------|-------------|
+| `Width`  | `string` | Controls the width of the Form. |
+| `Columns`  | `int` | Defines the number of columns in the Form. See the [Columns]({%slug form-columns%}) article for more information |
+| `ColumnSpacing`  | `string` | Defines the amout of vertical space between the Columns. See the [Columns]({%slug form-columns%}) article for more information. |
+| `Orientation`  | `FormOrientation` enum <br /> (`Vertical`) | controls the orientation of the Form. See the [Orientation]({%slug form-orientation%}) article for more information. |
 
-* `FormGroups` - lets you define Groups for the FormItems. See the [FormGroups]({%slug form-formgroups%}) article for more information.
+## Form Reference
 
-* `Orientation` - `enum` - controls the orientation of the Form. See the [Orientation]({%slug form-orientation%}) article for more information.
-    
-* `Columns` - `int` - defines the number of columns in the Form. See the [Columns]({%slug form-columns%}) article for more information.
+Use the Form Reference to get access to the `EditContext` object that the form will generate when you pass a `Model` to it. It could be useful to, for example, re-attach validation when you change the model - `FormReference.EditContext.AddDataAnnotationsValidation()`.
 
-* `ColumnSpacing` - `string` - defines the space between the FormItems. See the [Columns]({%slug form-columns%}) article for more information.
+>caption Get a reference to the Telerik Form for Blazor 
 
-* `Id` - `string` - sets an `id` attribute to the `<form>` element. It is possible to use it together with the [`Form` parameter of a submit button]({%slug button-type%}). Set both parameters to the same `string` value. This allows submitting the form from a button, which is outside the form.
+````CSHTML
+@* Get a reference to the Form component *@
+
+<TelerikForm Model="@person" @ref="@FormReference">
+</TelerikForm>
+
+@code {
+    public Telerik.Blazor.Components.TelerikForm FormReference { get; set; }
```

Comment:
This snippet does not compile currently. Either add the Person class, or make it non-runnable.

```suggestion
    private TelerikForm FormReference { get; set; }
```